### DISCUSSION TITLE
are you open to suggestions?

### DISCUSSION THREAD

**Discussion by Globalfirestorm10** [MartinTheDragon/Nuclear-Tech-Mod-Remake#14] [Q&A] [answered]
Hey Martin, im really glad you are porting HBM NTM to 1.18, are you open to suggestions, or do you hate them like bob does?

**Comment by MartinTheDragon** [comment, marked as answer]
Hi, I'll just copy and paste the answer from the FAQ channel on my discord: I might consider adding additional features when the time comes, but I also have some ideas of my own. (If it actually is a good idea, sure, I can add that in)

**Comment by Globalfirestorm10** [comment]
ok, thanks for answering martin :)
